### PR TITLE
Fix copy-paste error at Mappings.multiply

### PR DESCRIPTION
**Pull request by viliam-durina**

### CODE REVIEW

**Comment by danny0405** [comment]
Thanks for the contribution @viliam-durina , usually Calcite commit message does not start with "Fix", there are some typos mistakes there and we may merge it into bigger PRs.

**Comment by pivovarit** [comment]
Well, others made it. For example: https://github.com/apache/calcite/commit/fdcb195b829fdc4f52d777ae09630ee65eb0a977

**Review by danny0405 — APPROVED** [review]
+1, thanks for the changes.

**Comment by viliam-durina** [comment]
@danny0405 I don't know what to replace it with, "Repair", "Correct"? Couple of other commit msgs start with "fix".

File: core/src/main/java/org/apache/calcite/util/mapping/Mappings.java [hunk 1]
```diff
@@ -145,8 +145,8 @@ public static Mapping divide(Mapping mapping1, Mapping mapping2) {
   /**
    * Multiplies one mapping by another.
    *
-   * <p>{@code divide(A, B)} returns a mapping C such that B . C (the mapping
-   * B followed by the mapping C) is equivalent to A.
+   * <p>{@code multiply(A, B)} returns a mapping C such that A . B (the mapping
+   * B followed by the mapping A) is equivalent to C.
```
Comment (liyafan82) [inline core/src/main/java/org/apache/calcite/util/mapping/Mappings.java:149]: It should be "the mapping A followed by the mapping B"?

**Comment by danny0405** [comment]
> @danny0405 I don't know what to replace it with, "Repair", "Correct"? Couple of other commit msgs start with "fix".

No worries, would fix the commit when merge.

**Review by liyafan82 — APPROVED** [review]
LGTM

**Comment by julianhyde** [comment]
Thanks for the PR, @viliam-durina!

@viliam-durina, The main reason for the "no fix" rule is that people tend to create JIRA cases whose title is "Fix problem xxx" whereas the title should be "There's a problem with xxx".

Thanks for enforcing the rule, @danny0405. It makes better release notes for everyone.

**Pull request closed without merging**